apps/etterlatte-gyldig-soeknad/src/main/kotlin/gyldigsoeknad/omstillingsstoenad/GyldigOmstillingsSoeknadService.kt

```diff
@@ -0,0 +1,25 @@
+package no.nav.etterlatte.gyldigsoeknad.omstillingsstoenad
+
+import no.nav.etterlatte.gyldigsoeknad.client.PdlClient
+import no.nav.etterlatte.libs.common.behandling.Persongalleri
+import no.nav.etterlatte.libs.common.gyldigSoeknad.GyldighetsResultat
+import no.nav.etterlatte.libs.common.gyldigSoeknad.VurderingsResultat
+import no.nav.etterlatte.libs.common.soeknad.dataklasser.omstillingsstoenad.Omstillingsstoenad
+import java.time.LocalDateTime
+
+class GyldigOmstillingsSoeknadService(private val pdlClient: PdlClient) {
+    fun hentPersongalleriFraSoeknad(soeknad: Omstillingsstoenad): Persongalleri {
+        // TODO Må tilpasse persongalleri eller bruke noe annet?
```

Comment:
Jira?

Reply:
Har vel egentlig ikke en spesifikk jira for dette enda men mulig jeg oppretter en ny Persongalleri for OMS i EY-1777 som jeg jobber med nå. Da skal jeg tilpasse her.